### PR TITLE
[REVIEW] Spark-Proto Release

### PR DESCRIPTION
**Pull request by alachaum**

### CODE REVIEW

**Comment by alachaum** [comment]
@x4d3 could you review? thx.

File: ansible/group_vars/all [hunk 1]
```diff
@@ -1058,6 +1058,59 @@ rundeck:
     ttl: 60
     overwrite: yes
 
+# Spark Prototype configuration
+spark_proto:
+  skip: true
+  elastic_load_balancer:
+    name: "{{ vpc_short_name }}-sparkpto-elb"
+    connection_draining_timeout: 600
```
Comment (x4d3) [inline ansible/group_vars/all:1066]: Just by curiosity, why is it 60 for all the others except impac, connec and sparkpost ?

Comment (alachaum) [inline ansible/group_vars/all:1066]: Just a design choice related to closing connections on deregistering instances. We have a fairly generous connection draining on all components that are likely to have long running connections (e.g.: reporting). 600 is a little bit extreme though and should be lowered.

http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/config-conn-drain.html

**Comment by x4d3** [comment]
LGTM.

**Pull request closed without merging**